tests/ramping_load.test.yaml

```diff
@@ -0,0 +1,23 @@
+# This test ramps the load up in 3 steps throughout the test duration
+# with equal distribution of users, events, and sources
+
+# Remark: For the tests to be executed successfully,
+# the .env file needs to contain 6 sources.
```

Comment:
Can we mention in the comment that instead of having 6 sources, we can configure `HOT_SOURCES`?
Same in other test files.

Reply:
Removed the check for the instance index vs the source list and updated the tests. Now all replicas will handle the write keys only depending on the sources configured. The hot sources configuration can be used but is not necessary.